hls-test-utils/src/Test/Hls/Util.hs

```diff
@@ -322,38 +314,31 @@ expectNoMoreDiagnostics timeout doc src = do
 -- If timeout is 0 it will wait until the session timeout
 waitForDiagnosticsFromSourceWithTimeout :: Seconds -> TextDocumentIdentifier -> String -> Test.Session [Diagnostic]
 waitForDiagnosticsFromSourceWithTimeout timeout document source = do
```

Comment:
Previously, this function relied on logs to work. But the current test logger doesn't send logs to lsp client. I found a very similar function in `ghcide` which doesn't rely on logs, so changed this function accordingly.

Reply:
Right, but why are you sending `(A.toJSON GetFilesOfInterest)`. Seems like `A.Null` would express our intent more clearly?

Reply:
Yes, `A.Null` is better, and is working!